dev-vcs/gitolite-gentoo/gitolite-gentoo-3.6.6.1-r2.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+[[ ${PV} == *9999 ]] && SCM="git-2"
+EGIT_REPO_URI="git://git.gentoo.org/proj/gitolite-gentoo"
+EGIT_MASTER=master
+
+inherit perl-module user versionator ${SCM}
+
+DESCRIPTION="Highly flexible server for git directory version tracker, Gentoo fork"
+HOMEPAGE="https://cgit.gentoo.org/proj/gitolite-gentoo.git"
+if [[ ${PV} != *9999 ]]; then
+	SRC_URI="mirror://gentoo/${P}.tar.bz2"
+	KEYWORDS="~amd64 ~x86"
+else
+	SRC_URI=""
+	KEYWORDS=""
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="selinux tools vim-syntax"
+
+DEPEND="
+	acct-group/git
+	acct-user/git[gitolite]
+	dev-lang/perl
+	virtual/perl-File-Path
+	virtual/perl-File-Temp
+	>=dev-vcs/git-1.6.6"
+RDEPEND="${DEPEND}
+	!dev-vcs/gitolite
+	!www-apps/gitea
```

Comment:
what about introducing an `acct` use-flag and do blocking on gitolite/gitea only when this flag is enabled (as an advanced option - disable `acct-*` and skip home + user creation).

advantages:
- parallel install possible
- admin can choose the username (e.g gitolite for gitolite and gitea for gitea)
- admin can choose the home dir (e.g /srv instead of /var/lib)